src/invidious/yt_backend/extractors.cr

```diff
@@ -642,8 +652,13 @@ module HelperExtractors
   #
   # Returns a 0 when it's unable to do so
   def self.get_video_count(container : JSON::Any) : Int32
+    puts container
```

Comment:
Seems like a missing debug print ^^

Reply:
Sorry, mistake on my part. Should be fixed.